SPECS/vim/vim.spec

```
@@ -1,16 +1,15 @@
 %define debug_package %{nil}
 Summary:        Text editor
 Name:           vim
-Version:        8.2.3489
+Version:        8.2.3564
 Release:        1%{?dist}
 License:        Vim
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/Editors
 URL:            https://www.vim.org
-#Source0:       https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz
+#Source0:       https://github.com/%%{name}/%%{name}/archive/v%%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
```

What you could also do:

```suggestion
Source0:       https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

This way:
- `rpmbuild` still gets the right file name.
- It's easy to retrieve the URL when parsing with `rpmspec`, since the URL is no longer in a comment ignored by the tool.
- It's just one line.

---

Suggestion taken, thanks!